src/lib/components/dropdown/Dropdown.component.tsx

```diff
@@ -81,25 +59,36 @@ const DropdownMenuItemStyled = styled.li`
   white-space: nowrap;
   cursor: pointer;
   font-size: ${fontSize.base};
+  ${(props) => {
+    console.log(props.isSelected);
+    return props.isSelected
+      ? `background-color: ${props.theme.highlight};`
+      : `background-color: ${props.theme.backgroundLevel1};`;
+  }}
+
+  color: ${getThemePropSelector('textPrimary')};
+  border-top: 0.3px solid ${getThemePropSelector('border')};
```

Comment:
Should we make this value of "0.3px" a variable/token, to have consistency across components?

Reply:
I think this value is specific to the dropdown component, not sure it's really useful, is it?